lemur/sources/service.py

```diff
@@ -198,7 +198,7 @@ def clean(source):
 
     try:
         certificates = s.clean(source.options)
-    except NotImplemented:
+    except NotImplementedError:
```

Comment:
You are correct I don't believe this is a hot code path so I probably missed this.